package.json

```diff
@@ -13,10 +13,11 @@
     "groq": "bin/groq.js"
   },
   "engines": {
-    "node": ">=10"
+    "node": ">=18"
   },
   "scripts": {
-    "test": "eslint ."
+    "test": "echo '[{\"foo\": 1}]' | ./bin/groq.js --pretty '*[]{ foo }'",
```

Comment:
Nit: Shall we move it out of package.json and to a file instead?

Reply:
Ideally I would want to create some proper tests 😅 This was just the path of least resistance for now